site/docs/components/slider/accessibility.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+## Cursor interaction
+
+Allow cursor interaction of a handle by increasing the target area around the indicator to -size-base. This creates a more accessible click/touch area which meets WCAGs minimum Target Size (2.5.5) standard.
```

Comment:
This doesn't read as a11y guidance to our consumers. It either needs to be re-worded or removed. cc @navkaur76